.gitignore

```diff
@@ -59,6 +59,7 @@ __pycache__
 *.pyc
 *.swp
 \.cache
+.devcontainer/
```

Comment:
VS Code has an option to do your actual development inside of a Docker container so that you have a repeatable dev environment. The `.devcontainer/` directory is where VS Code stores the files related to building and customizing your dev container.

There is no privacy or security issue with adding this directory to source control, I added it to .gitignore because I didn't think anyone else develops inside of a Docker container.